src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -19,7 +19,10 @@
 import java.io.*;
 import java.net.*;
 import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
```

Comment:
Could you please use package imports (according to the Jitsi Code Convention)? Thank you!